google/detectors/credentials/ncrack/src/main/java/com/google/tsunami/plugins/detectors/credentials/ncrack/client/NcrackClient.java

```diff
@@ -160,23 +164,47 @@ String getFlag() {
   private ImmutableList<String> passwordList;
   private TargetService targetService;
 
+  @Parameters(separators = "=")
+  public static class NcrackClientCliOptions implements CliOption {
```

Comment:
Please add a Javadoc comment for this class.

Reply:
added javadoc